x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/alerts/error_count_threshold.spec.ts

```diff
@@ -50,7 +51,14 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       ],
     };
 
-    before(() => {
+    before(async () => {
+      supertestWithRoleScope = await roleScopedSupertest.getSupertestWithRoleScope('viewer', {
+        withInternalHeaders: true,
+        useCookieHeader: true,
+      });
+
+      roleAuthc = await samlAuth.createM2mApiKeyWithRoleScope('admin');
```

Comment:
Ok, so we've got the supertest above looking good. 
Now, why is this one admin?  Can it not be lower?

Reply:
alerting api apparently needs `admin` role to manually create the info needed to run the test.

Reply:
Sometimes some indices are missing within https://github.com/crespocarlos/kibana/blob/3a56a72fc4d7ebff7f10fb9cd4a886b22e25ae21/packages/kbn-es/src/serverless_resources/project_roles/oblt/roles.yml#L40

Is that the case here?

Reply:
I can't really say for sure. Also, all alerting tests are using `admin` role. I'd rather follow what they are doing for now.